harness/determined/tensorboard/base.py

```diff
@@ -36,20 +44,16 @@ def list_tfevents(self) -> List[pathlib.Path]:
             )
             return []
 
-        return [f.resolve() for f in self.base_path.glob("**/*tfevents*")]
+        def stat_each(paths):
+            for f in paths:
+                yield (f, f.stat())
 
-    def to_sync(self) -> List[pathlib.Path]:
-        """
-        to_sync returns tfevent files that have not been exported to the persistent
-        storage backend.
-        """
+        return [f for f, s in stat_each(self.base_path.glob("**/*tfevents*")) if s.st_mtime > since]
```

Comment:
Haven't tested it, but I believe this can replace the several lines above.

```python
        return [f for f in self.base_path.glob("**/*tfevents*") if f.stat().st_mtime > since]
```

Reply:
oh right!  Originally I thought I had to return the stat output too, which is why I introduced the extra layer of iterator (didn't want to stat twice)